cli_reference/developer_cli_odo/installing-odo.adoc

```diff
@@ -6,23 +6,23 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
+// The following section describes how to install `{odo-title}` on different platforms using the CLI or the Visual Studio Code (VS Code) IDE.
+
+You can install the `{odo-title}` CLI on Linux, Windows, or macOS by downloading a binary. You can also install the OpenShift VS Code extension, which uses both the `{odo-title}` and the `oc` binaries to interact with your OpenShift Container Platform cluster. For {op-system-base-full}, you can install the `{odo-title}` CLI as an RPM.
 
 [NOTE]
 ====
 Currently, `{odo-title}` does not support installation in a restricted network environment.
 ====
 
-You can also find the URL to the latest binaries from the {product-title} web console by clicking the *?* icon in the upper-right corner and selecting *Command Line Tools*
+// You can also find the URL to the latest binaries from the {product-title} web console by clicking the *?* icon in the upper-right corner and selecting *Command Line Tools*
```

Comment:
Once again, I don't see this in the preview:
<img width="1332" alt="Screen Shot 2022-01-19 at 2 41 14 PM" src="https://user-images.githubusercontent.com/70717303/150202124-a418adcb-2f6f-4862-822b-0d92af1d97f4.png">


The sentence is good, but for accessibility, we want to avoid words with directions, like top, bottom, left, right. So consider a simple rewrite:

`...web console by clicking the *7* icon and selecting *Command Line Tools*.`

Reply:
I have explicitly commented out this option, because it currently leads you to an older version of odo, namely 2.4.3 and to a different mirror, rather than the content gateway. Once with is updated in a newer release of OpenShift, I will restore the instructions.